sky/spot/spot_state.py

```diff
@@ -158,14 +162,15 @@ def failure_statuses(cls) -> List['SpotStatus']:
     SpotStatus.PENDING: colorama.Fore.BLUE,
     SpotStatus.SUBMITTED: colorama.Fore.BLUE,
     SpotStatus.STARTING: colorama.Fore.BLUE,
-    SpotStatus.RUNNING: colorama.Fore.GREEN,
+    SpotStatus.RUNNING: colorama.Fore.BLACK,
```

Comment:
Fore.BLACK => invisible for black console background?

Reply:
Ahh, good point! Reverted it.